client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/dynamic-settings.js

```diff
@@ -0,0 +1,75 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
Too bad we're using `react-redux@5.*`. If we were using `7.1`, we would have been able to use hooks also on this :)